modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,205 @@
+
+Effortless File Collection: With Secret File Drop, collecting files becomes a seamless process. Simply create a file drop link, share it with recipients, and let them submit files directly without any additional steps or complications. This feature saves time and eliminates the need for manual file collection methods, such as email attachments or physical hand-ins.
+
+[discrete]
+=== 3 View Modes
+
+Users can now choose three distinct view modes, enhancing file browsing and navigation options:
+
+* *Compact List View* +
+The Compact List View provides a condensed and space-efficient representation of your files and folders, designed for users who prefer a streamlined view.
+* *Regular List View* +
+The Regular List View offers a familiar and comprehensive approach to file organization.
+* *Tiles View* +
+The Tiles View introduces a visually appealing and intuitive way to browse your files and folders. In this mode, files and folders are displayed as colorful and resizable tiles, providing a visually engaging experience. Users can preview file contents and relevant details directly within the tiles, allowing for quick identification and navigation. This is particularly beneficial for users who prioritize visual recognition and prefer a visually rich interface.
+
+[discrete]
+=== Trash Bins for Spaces
+
+The Separate Trash Bin for Spaces introduces individual trash bins for each space within your organization's file system. A space represents a dedicated area where teams collaborate on specific projects, departments, or initiatives. With this feature, accidental file deletions are a thing of the past, as files deleted within a space are now moved to a separate trash bin specific to that space.
+
+[discrete]
+=== Deny Access
+The Deny Access feature, which is *experimental and not ready for production environments*, allows users to share folders with groups, but deny access to a single person in that group. Example: If you share the "Birthday Present" folder with all users in the organization, you want to exclude the one person whose birthday is coming up. This use case is now possible, but not yet production ready and disabled by default. To enable this feature, the [variable](https://doc.owncloud.com/ocis/next/deployment/services/s-list/frontend.html#environment-variables) `FRONTEND_OCS_ENABLE_DENIALS` needs to be set to to `true`. Please contact {oc-support-url}[ownCloud Support] or give us your feedback via {oc-central-url}[ownCloud Central] so that we can finalize this long-awaited feature with the help of your testing.
```

Comment:
```suggestion
The Deny Access feature, which is *experimental and not ready for production environments*, allows users to share folders with groups, but deny access to a single person in that group. Example: If you share the "Birthday Present" folder with all users in the organization, you want to exclude the one person whose birthday is coming up. This use case is now possible, but not yet production ready and disabled by default. To enable this feature, the https://doc.owncloud.com/ocis/next/deployment/services/s-list/frontend.html#environment-variables[variable] `FRONTEND_OCS_ENABLE_DENIALS` needs to be set to to `true`. Please contact {oc-support-url}[ownCloud Support] or give us your feedback via {oc-central-url}[ownCloud Central] so that we can finalize this long-awaited feature with the help of your testing.
```